.github/workflows/docker.yml

```
@@ -44,8 +43,7 @@ jobs:
         with:
           images: cloudamqp/lavinmq
           tags: |
-            type=raw,value=latest,enable={{is_default_branch}}
-            type=semver,pattern={{version}}
+            type=semver,pattern={{version}},value=latest
```

Actually, maybe we want to keep the `main` tag still:

```suggestion
            type=raw,enable={{is_default_branch}}
            type=semver,pattern={{version}},value=latest
```